libs/template/application/src/auth/bapi/feature.ts

```diff
@@ -34,10 +35,10 @@ export class BapiAuthFeature extends OauthFeature implements AppFeature {
           clientId: 'frontend',
           grantType: 'authorization_code',
           authUrl: new URL('/login', globalThis.location.origin).toString(),
-          tokenUrl: new URL(
-            '/token',
-            injectEnv('ORYX_FULFILLMENT_BACKEND_URL')
-          ).toString(),
+          tokenUrl: urlJoin(
```

Comment:
What is the benefit of using third-party package here?

Reply:
Because it doesn't get the right url. If you have a `ORYX_FULFILLMENT_BACKEND_URL` like this: `https://fulfillment-app.netlify.app/functions/server`, you end up with `https://fulfillment-app.netlify.app/token` using `new URL` instead of `https://fulfillment-app.neltify.app/functions/server/token`, which is what we want.